.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
It seems you pushed your VS code settings file and folder along with your change. Please remove it so that it does not show up among files changed @kivuvarosekivuvan 
#
> This is something you can always check for your self in the file changes here: https://github.com/ushahidi/platform-client-mzima/pull/1175/files

Reply:
@Ifycode Done ✅